react/src/lib/components/DeckGLMap/layers/map/mapLayer.ts

```diff
@@ -348,7 +348,7 @@ export default class MapLayer extends CompositeLayer<MapLayerProps<unknown>> {
                 meshLines: this.state["mesh_lines"],
                 pickable: this.props.pickable,
                 modelMatrix: rotatingModelMatrix,
-                contours: this.props.contours,
+                contours: !isMesh ? [-1, -1] : this.props.contours,
```

Comment:
Maybe we should consider the case where you want contours on a property, eg. thickness, even if there is no mesh. 

@Midtveit do you think it can be useful to have contours on properties?

Reply:
Yes that should be possible. I merged it already but I'll make a new issue on it